modules/cloud-run/README.md

```diff
@@ -0,0 +1,166 @@
+# Cloud Run Module
+
+Cloud Run management, with support for IAM roles and optional Eventarc trigger creation.
+
+## Examples
+
+### Traffic split
+
+This deploys a Cloud Run service with traffic split between two revisions.
+
+```hcl
+module "cloud_run" {
+  source     = "../../modules/cloud-run"
+  project_id = "my-project"
+  name       = "hello"
+  revision_name = "green"
+  containers = [{
+    image   = "us-docker.pkg.dev/cloudrun/container/hello"
+    command = null
+    args    = null
+    env     = null
+    env_from = null
+    ports = null
+    resources = null
+    volume_mounts = null
+  }]
+  traffic = {
+    "blue" = 25
+    "green" = 75
+  }
+}
+# tftest:skip
```

Comment:
Can you make sure documentation tests run?

Reply:
Done